podium/talks/templates/talks/session-content.html

```diff
@@ -0,0 +1,26 @@
+<div >
+  <h4>{{session.description}}</h4>
+</div>
+
+<!-- Table -->
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Speaker</th>
+      <th>Email</th>
+    </tr>
+  </thead>
+
+  <tbody>
+
+    {%for talk in talks%}
+    <tr>
+      <td><a href={{talk.get_absolute_url}}>{{talk.title}}</a></td>
+      <td>{{talk.speaker_name}}</td>
+      <td>{{talk.speaker_email}}</td>
```

Comment:
We should avoid exposing the speaker's email publicly without explicit permission.  Let's not include it on public-facing pages until we have an opt-in flag to check.